tests/console/firewalld.pm

```diff
@@ -0,0 +1,142 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test FirewallD basic usage
+# Maintainer: Alexandre Makoto Tanno <atanno@suse.com>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use transactional_system 'trup_install';
+
+sub run {
+    select_console("root-console");
+
+    # Check Service State, enable it if necessary
+    record_info 'Check Service State';
+    assert_script_run("if [ `firewall-cmd --state > /dev/null 2>&1 ; echo $?` -eq 252 ]; then systemctl start firewalld; fi");
```

Comment:
why not `systemctl is-active firewalld`  ?

Reply:
I can change this as well.

Reply:
I've modified the test to use `systemctl is-active firewalld`.